src/components/LoginPage/ForgotPassword/index.css

```diff
@@ -0,0 +1,17 @@
+.ressPassBtn {
+  background-color: #c62a88;
+  border: none;
+}
+
+.purple {
```

Comment:
Can you give more unique names for theses classNames, so the reader would understand what are they referring to

Reply:
PTAL